axlearn/common/checkpointer_orbax.py

```diff
@@ -187,15 +190,20 @@ class Config(BaseCheckpointer.Config):
 
         Attributes:
             keep_last_n: Keep this many past ckpts.
+            keep_every_n_steps: If set, keep a checkpoint every n steps.
```

Comment:
Given that OrbaxEmergencyReplicatorCheckpointer inherits from BaseCheckpointer (not from this one), are these changes related to MTC integration?

Reply:
nope -- removed